NewForce-Cohort-6/tabloidfullstack-mashed-potatoes
# 21DeleteComment

# Description
[Trello Ticket #21 (https://trello.com/c/34dPhzQJ)]

Added feature to delete a comment from a post. The comment can be deleted by clicking the delete comment button at the bottom of the comment. This will take you to a page asking if you're sure you want to delete the comment. This page will have a delete button to confirm deleting the comment and a link to cancel the delete by returning you to the comment.

# Testing Instructions
1. `git add` and `git commit` whatever you're working on on your branch
2. `git checkout main`
3. `git fetch --a`
4. `git checkout 20CreateComment`
5. If you don't see the most recent version of my files, `git pull origin 19ViewPostComments'
6. If you haven't created the Tabloid database, please do so.
Use this data to fill out tables and columns: [SQL Tables](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/01_Tabloid_Create_DB.sql)
Use this to seed the database : [SQL SEED DATA](https://github.com/NewForce-Cohort-6/tabloidfullstack-mashed-potatoes/blob/main/SQL/02_Tabloid_Seed_Data.sql)
7. Click the execute with debugger solid green button to launch the app
8. Log into the website using “leo@gmail.com” if greeted with a login page
9. From the home page, click the “Post or My Post” option in the navbar
10. A list of posts will show on the page.
11. Choose a post to view. This will take you to the post details page with options to view comments, return to the post list or add a comment at the bottom of the page.
12. Click the view comment link. This will take you to a list of comments on the post. 
13. The bottom of each comment will have a delete button. Click the delete button which will take you to a page asking if you're sure you want to delete the comment including a delete button and a back to comments link.
14. First click the back to comments link to return you to the comment list.
15. Lastly, click the delete button on the comment again. On the delete comment page, click the delete button. This will delete the comment and return you to the updated comment list.



# Checklist:
